arctouch-challenge-ios/Models/Movie.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  Movie.swift
+//  arctouch-challenge-ios
+//
+//  Created by Pedro H J Fabrino on 10/03/18.
+//  Copyright © 2018 AIS Digital. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+class Movie: Mappable {
```

Comment:
I would use the Codale protocol, which includes encoding and decoding. The reason I choose to use ObjectMapper is for being able to work with dictionaries, instead of Data objects.